sctp.h

```diff
@@ -74,6 +74,8 @@ void janus_sctp_deinit(void);
 #define DATA_CHANNEL_FLAGS_SEND_RSP 0x00000002
 #define DATA_CHANNEL_FLAGS_SEND_ACK 0x00000004
 
+struct janus_ice_handle;
```

Comment:
By using forward declaration here instead of including `ice.h`, all files only including `sctp.h` but not `ice.h` don't need to get recompiled if `ice.h` changes. This helps speeding up compilation and avoiding unnecessary recompilation.